sql-statements/sql-statement-kill.md

```diff
@@ -39,6 +39,12 @@ Query OK, 0 rows affected (0.00 sec)
 * By design, this statement is not compatible with MySQL by default. This helps prevent against a case of a connection being terminated on the wrong TiDB server, because it is common to place multiple TiDB servers behind a load balancer.
 * The `KILL TIDB` statement is a TiDB extension. If you are certain that the session you are attempting to kill is on the same TiDB server, set [`compatible-kill-query = true`](/tidb-configuration-file.md#compatible-kill-query) in your configuration file.
 
+## Global Kill since TiDB v5.0
+
+Since v5.0, TiDB provide a feature call [global kill](https://github.com/pingcap/tidb/blob/master/docs/design/2020-06-01-global-kill.md), to enable safely killing any connection, even when TiDB servers are behide a load balancer.
+
+To enable "global kill", set `enable-global-kill = true` in `experimental` section of your configuration file.
```

Comment:
```suggestion
Since v5.0, TiDB provides experimental support for [global kill](https://github.com/pingcap/tidb/blob/master/docs/design/2020-06-01-global-kill.md). When enabled, each TiDB server will ensure that Connection IDs are globally unique. A `KILL` statement can be issued to any TiDB server, which will internally route the request to the correct TiDB instance. This ensures that `KILL` is safe even when TiDB servers are behind a load balancer.

To enable "global kill", set `enable-global-kill = true` in `experimental` section of your configuration file.
```